src/Api/HttpRestClient.php

```diff
@@ -0,0 +1,365 @@
+<?php
+
+declare(strict_types=1);
+
+namespace KafkaRest\Api;
+
+use Assert\Assertion;
+use Http\Discovery\Psr17FactoryDiscovery;
+use Http\Discovery\Psr18ClientDiscovery;
+use JMS\Serializer\SerializerBuilder;
+use JMS\Serializer\SerializerInterface;
+use KafkaRest\Api\Exception\UnexpectedApiResponse;
+use KafkaRest\Api\Value\Request\ConsumerOptions;
+use KafkaRest\Api\Value\Request\Subscription;
+use KafkaRest\Api\Value\Response\AssignedPartition;
+use KafkaRest\Api\Value\Response\Consumer;
+use KafkaRest\Api\Value\Response\Message;
+use KafkaRest\Api\Value\Response\Offset;
+use KafkaRest\Api\Value\Response\Partition;
+use KafkaRest\Api\Value\Response\ProduceResults;
+use KafkaRest\Api\Value\Response\Topic;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestFactoryInterface;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\StreamFactoryInterface;
+use Psr\Http\Message\UriFactoryInterface;
+use Psr\Http\Message\UriInterface;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Teapot\StatusCode\Http;
+use function http_build_query;
+use function Safe\json_decode;
+use function Safe\sprintf;
+
+final class HttpRestClient implements RestClient
+{
+    /** @var LoggerInterface */
```

Comment:
abcd sort

Reply:
it's sorted as params in the constructor